scalahost/src/main/scala/scala/meta/internal/scalahost/ScalahostPipeline.scala

```diff
@@ -13,11 +13,11 @@ trait ScalahostPipeline { self: ScalahostPlugin =>
 
   object ScalahostComponent extends PluginComponent {
     val global: ScalahostPipeline.this.global.type = ScalahostPipeline.this.global
-    val runsAfter                                  = List("typer")
-    override val runsRightAfter                    = Some("typer")
-    val phaseName                                  = "scalameta"
-    override val description                       = "compute the scala.meta semantic database"
-    def newPhase(_prev: Phase)                     = new ScalahostPhase(_prev)
+    val runsAfter = List("typer")
+    override val runsRightAfter = Some("typer")
+    val phaseName = "scalameta"
+    override val description = "compute the scala.meta semantic database"
+    def newPhase(_prev: Phase) = new ScalahostPhase(_prev)
```

Comment:
I'm glad you like it. Please continue reporting what you dislike with scalafmt. 

It might be useful to configure a git hook to run scalafmt before committing.